pkgs/by-name/me/meson/package.nix

```diff
@@ -21,13 +21,13 @@ let
 in
 python3.pkgs.buildPythonApplication rec {
   pname = "meson";
-  version = "1.4.1";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "mesonbuild";
     repo = "meson";
     rev = "refs/tags/${version}";
-    hash = "sha256-RBE4AUF5fymUA87JEDWtpUFXmVPFzdhZgDI7/kscTx4=";
+    hash = "sha256:1gzibd84klvndqvamrjj8pfwsslk84pgkq4vjai1yxsdf1yzrzv3";
```

Comment:
Update your hasher :)

Reply:
Wow that's new to me, where are the docs for this 😂